doc/source/schemas/sequenceAnnotationmethods.rst

```diff
@@ -0,0 +1,457 @@
+SequenceAnnotationMethods
+*************************
+
+ .. function:: searchFeatureSets(request)
+
+  :param request: SearchFeatureSetsRequest: This request maps to the body of `POST /featuresets/search` as JSON.
+  :return type: SearchFeatureSetsResponse
+  :throws: GAException
+
+Gets a list of `FeatureSet` matching the search criteria.
+
+  `POST /featuresets/search` must accept a JSON version of
+  `SearchFeatureSetsRequest` as the post body and will return a JSON version
+  of `SearchFeatureSetsResponse`.
+
+ .. function:: getFeatureSet(id)
+
+  :param id: string: The ID of the `FeatureSet`.
+  :return type: org.ga4gh.models.FeatureSet
+  :throws: GAException
+
+Gets a `FeatureSet` by ID.
+  `GET /featuresets/{id}` will return a JSON version of `FeatureSet`.
+
+ .. function:: getFeature(id)
+
+  :param id: string: The ID of the `Feature`.
+  :return type: org.ga4gh.models.Feature
+  :throws: GAException
+
+Gets a `org.ga4gh.models.Feature` by ID.
+  `GET /features/{id}` will return a JSON version of `Feature`.
+
+ .. function:: searchFeatures(request)
+
+  :param request: SearchFeaturesRequest: This request maps to the body of `POST /features/search` as JSON.
+  :return type: SearchFeaturesResponse
+  :throws: GAException
+
+Gets a list of `Feature` matching the search criteria.
+
+  `POST /features/search` must accept a JSON version of
+  `SearchFeaturesRequest` as the post body and will return a JSON version of
+  `SearchFeaturesResponse`.
+
+.. avro:enum:: Strand
+
+  :symbols: NEG_STRAND|POS_STRAND
+  Indicates the DNA strand associate for some data item.
+  * `NEG_STRAND`: The negative (-) strand.
+  * `POS_STRAND`:  The postive (+) strand.
+
+.. avro:record:: Position
+
+  :field referenceName:
+    The name of the `Reference` on which the `Position` is located.
+  :type referenceName: string
+  :field position:
+    The 0-based offset from the start of the forward strand for that `Reference`.
+      Genomic positions are non-negative integers less than `Reference` length.
+  :type position: long
+  :field strand:
+    Strand the position is associated with.
+  :type strand: Strand
+
+  A `Position` is an unoriented base in some `Reference`. A `Position` is
+  represented by a `Reference` name, and a base number on that `Reference`
+  (0-based).
+
+.. avro:record:: ExternalIdentifier
+
+  :field database:
+    The source of the identifier.
+      (e.g. `Ensembl`)
+  :type database: string
+  :field identifier:
+    The ID defined by the external database.
```

Comment:
@reece,  the sky *is* falling.   Early attempts to more formalize the data semantics were rejected with the notation that the API provides the mechanism and the data semantics can be layered on top.  That said, we should maybe sure we can represent the semantics.